aeon/dj_pipeline/streams.py

```diff
@@ -15,17 +15,15 @@
 
 logger = dj.logger
 
-
-schema_name = f'u_{dj.config["database.user"]}_streams'  # for testing
-# schema_name = get_schema_name("streams")
+schema_name = get_schema_name("streams")
```

Comment:
Is this fully tested and ready for production deployment?

Reply:
not with `presocial` yet..  but there tables being created under `aeon_test_streams` by the worker.